docs/getting-started/overview.rst

```diff
@@ -1,6 +1,92 @@
-.. include:: /stub-topic.txt
+Overview of .NET implementations
+================================
+
+.NET Platform is, at its very core, a set of standards that can be implemented. There are 
+various implementations, some coming from Microsoft, some coming from other companies and 
+groups. This document will outline the current state of the .NET ecosystem and its various 
+implementations, so you can get a good "lay of the land". 
+
+.NET Framework
+--------------
+
+The .NET Framework is the premier implemetentation of the .NET Platform available for 
+Windows server and client developers. It is a very powerful, very mature framework, with 
+a huge class library (known as the **Base Class Library** or BCL for short) that supports 
+a wide variety of applications and solutions on Windows. 
+
+The additional stacks built on top the .NET Framework allow developers to build software 
+from CLI tools to rich web applications. 
+
+**Windows Forms** and **Windows Presentation Foundation** are two stacks that allow developers 
+to build desktop applications for Windows operating system. The former is focused more on LOB 
+apps, while the latter allows developers great control over the layout and provides opportunities
+to create really stunning user interfaces.
+
+**ASP.NET** is Microsoft's main web platform that runs on .NET Framework (among other things). 
```

Comment:
I'm not sure what you mean by "main". What's the other one?

What you mean by "(among other things)"? I'm not sure that's needed.

I'd suggest talking about the various pieces of ASP.NET: Web Forms, MVC, Web API and Signal R. While on one level, they are just libraries, on the other, they offer very different experiences.

WCF is missing.